src/mscorlib/ILLinkTrim.xml

```diff
@@ -19,5 +19,7 @@
       <!-- Methods is used by VS Tasks Window. -->
       <method name="GetActiveTaskFromId" />
     </type>
+    <!-- Accessed via private reflection by tracing controller. -->
+    <type fullname="System.Diagnostics.Tracing.EventPipe*" />
```

Comment:
@brianrob I am adding EventPipe to the exclusion list for ILLinker, so that it is not getting stripped. I think this exclusion should be removed once #15375 is fixed.

Reply:
Thanks @jkotas.  Agreed.